nodes/scheduling/secondary_scheduler/index.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="nodes-secondary-scheduler-about"]
+= Secondary scheduler overview
+include::_attributes/common-attributes.adoc[]
+:context: nodes-secondary-scheduler-about
+
+toc::[]
+
+You can install the {secondary-scheduler-operator} to run a custom secondary scheduler alongside the default scheduler to schedule pods.
+
+// About the {secondary-scheduler-operator}
+include::modules/nodes-secondary-scheduler-about.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_nodes-secondary-scheduler-about"]
+== Additional resources
+
+* xref:../../../nodes/scheduling/nodes-scheduler-about.adoc#nodes-scheduler-about[Controlling pod placement using the scheduler].
```

Comment:
@bergerhoffer do we really need this ?

Reply:
Good catch, I think I removed the mention of the default scheduler from the note, but not this additional resources link. Removed!